### DISCUSSION TITLE
Helper function to get string_view of enums.

### DISCUSSION THREAD

**Discussion by jbbjarnason** [stephenberry/glaze#551] [Ideas]
When using glz::enumerate ,I would like to get a constexpr std::string_view of the enum transpose. 

Specifically this part of glaze
```cpp
  // from json read.hpp 
  static constexpr auto frozen_map = detail::make_string_to_enum_map<T>();
  const auto& member_it = frozen_map.find(key);
  if (member_it != frozen_map.end()) {
     value = member_it->second;
  }
  else [[unlikely]] {
     ctx.error = error_code::unexpected_enum;
  }
```

Would it make sense to make such a function in glz namespace? (as in not `glz::detail`)

**Comment by stephenberry** [comment]
Yeah, go ahead and move it out of detail if you'd like to use it.

**Reply by jbbjarnason** [reply]
glz::enum_name_v was actually what I needed, so no changes needed.
